JenkinsConsoleUtility/JenkinsScripts/testInit.sh

```diff
@@ -11,75 +11,45 @@ CheckVerticalizedParameters
 CheckDefault SHARED_WORKSPACE C:/depot
 CheckDefault WORKSPACE C:/proj
 
-# USAGE: DelArcPatches <FullPath>
-DelArcPatches (){
-    echo === DelArcPatches $PWD, $@ ===
-    cd "$1"
-    echo Deleting Arc-Patches in: ${PWD}
-    git for-each-ref --format='%(refname:short)' refs/heads/arcpatch* | while read branch; do    BRANCH_EXISTS=$( git ls-remote --heads origin $branch | wc -l );    if [ $BRANCH_EXISTS -eq 0 ]; then        git branch -D $branch;    fi;done
-}
-
-# USAGE: ApplyArcPatch
-ApplyArcPatch (){
-    if [ -n "$PatchRepoName" ] && [ "$PatchRepoName" = "pf-main" ]; then
-        cd "$WORKSPACE/$PatchRepoName"
-        echo ==== arc patching pf-main ====
-        arc patch $DIFF_NUMBER --conduit-token $JENKINS_PHAB_TOKEN
-        echo ==== applyArcPatch Done ====
-    fi
-    if [ -n "$PatchRepoName" ] && [ "$PatchRepoName" = "SDKGenerator" ]; then
-        cd "$WORKSPACE/$PatchRepoName"
-        echo ==== arc patching SDKGenerator ====
-        arc patch $DIFF_NUMBER --conduit-token $JENKINS_PHAB_TOKEN
-        echo ==== applyArcPatch Done ====
-    fi
-    if [ -n "$PatchRepoName" ] && [ "$PatchRepoName" = "SdkName" ]; then
-        cd "$WORKSPACE/sdks/$PatchRepoName"
-        echo ==== arc patching $SdkName ====
-        arc patch $DIFF_NUMBER --conduit-token $JENKINS_PHAB_TOKEN
-        echo ==== applyArcPatch Done ====
-    fi
-}
-
-DoNugetWork (){
+DoJcuNugetUpdate (){
     pushd "$WORKSPACE/SDKGenerator/JenkinsConsoleUtility"
     cmd <<< "nuget restore JenkinsConsoleUtility.sln"
     popd
-    # None of the SDKs need pf-main directly
-    # pushd "$WORKSPACE/pf-main/Server"
-    # cmd <<< "nuget restore Server.sln"
-    # popd
+}
+
+# USAGE: ResetRepo
+ResetRepo (){
+    echo === ResetRepo $PWD, $@ ===
+
+    # Assumes the current directory is set to the repo to be reset
+    SetGitHubCreds
```

Comment:
I see this call was here before, but I don't see a function declaration. What is this line in particular doing?